config/routes.rb

```diff
@@ -3,4 +3,10 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+
+
+  get "/customers/:id", to: "customers#show"
+  patch "/customers/:id", to: "customers#update"
```

Comment:
As mentioned in the controller, this is a more RESTful option for the form functionality:
`post "/customers/:id/items", to: "customer_items#create"`

Note change in path *and* different controller